app/assets/javascripts/qsets.js

```diff
@@ -41,6 +41,7 @@ function initQuestionDisplayModal($modal, $question_link) {
 
   initAnswerButton();
   initUserFeedback();
+  initVote();
```

Comment:
(So @lizamcpherson01 looks like this line can be removed -- the only one you need moving forward is [the one you already have in the questions/show page](https://github.com/penzance/askup/pull/83/files#diff-e6ffad0524f0950ee6018b3f68e43342R67).)